docs/js-guide.md

```diff
@@ -103,6 +103,36 @@ function getPDFTitle() {
 class PDFIntegration { ... };
 ```
 
+#### Ref naming
+
+Preact does not allow the `ref` prop to be passed verbatim on non-HTML components. Instead it
+offers a wrapper method called `forwardRef` to extract the ref for you. Unfortunately the wrapper
```

Comment:
Can you make `forwardRef` a link to the relevant page in the Preact docs: https://preactjs.com/guide/v10/switching-to-preact/#forwardref.